css/at-rules/import.json

```diff
@@ -7,10 +7,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@import",
           "support": {
             "chrome": {
-              "version_added": true
+              "version_added": "2"
```

Comment:
Your description says `@import` is version 1 for Chrome and Safari — neither lines up here. Can you shed some light on this?

Reply:
Whoops!  That would have to be a typo, fixing that now.  :P